doc/guides/security-release-process.md

```diff
@@ -38,9 +38,10 @@ information described.
   * Described in the pre/post announcements
 
 * [ ] Pre-release announcement [email][]: ***LINK TO EMAIL***
+   * CC: `oss-security@lists.openwall.com`
+   * Subject: `Security updates for all active release lines, Month Year`
```

Comment:
Should we specify 'Node.js security updates' in the subject line for clarity?

Reply:
Sounds good to me, I'll update shortly.